main.js

```diff
@@ -1,14 +1,28 @@
-const { app, BrowserWindow } = require("electron");
+import { app, BrowserWindow, Tray, Menu, nativeImage } from "electron";
+import { fileURLToPath } from 'url';
+import path, { dirname } from 'path';
+
+let win;
+let tray;
 
 const createWindow = () => {
```

Comment:
this function might return window and we can use it inside `app.whenReady().then(() => {` 
obviously don't need to save variable top of this file